### ISSUE TITLE
Add a smoke test script

### ISSUE THREAD

**Issue by schu** [kinvolk/kubeadm-nspawn#56]
The script, for instance `./tests/smoke.sh`, would

* do a clean kubeadm-nspawn Vagrant setup,
* use kubectl to verify the successful setup of k8s,
* do a deploy and
* query the deployed service to verify it's running.

**Comment by robertgzr** [comment]
To check k8s setup?
```kubectl -n kube-system get nodes```

What can we deploy to test?
https://github.com/kubernetes/kubernetes/tree/master/examples/https-nginx ?

**Comment by robertgzr** [comment]
Or maybe https://github.com/kubernetes/kubernetes/tree/master/examples/guestbook

**Comment by schu** [comment]
> To check k8s setup?

Yes, a minimal script to test everything (that is expected to work) works as expected.
